app/controllers/admin/rewards_controller.rb

```diff
@@ -3,7 +3,7 @@
 module Admin
   class RewardsController < AdminController
     def index
-      @rewards = Reward.all
+      @rewards = Reward.all.order(:title)
```

Comment:
I think there is an N+1 query with the attachments - please check it out
```suggestion
      @rewards = Reward.all.with_attached_photo.order(:title)
```

Reply:
true, i forgot to check rewards in admin panel for n+1